charts/gardener/controlplane/charts/runtime/templates/apiserver/configmap-admission-config.yaml

```
@@ -1,4 +1,5 @@
-{{- if and .Values.global.apiserver.enabled .Values.global.apiserver.admissionConfig }}
+{{- if and .Values.global.apiserver.enabled (include "gardener-apiserver.hasWebhookTokens" .) }}
```

Can you please double check if it should rather be `hasAdmissionPlugins`?
```suggestion
{{- if and .Values.global.apiserver.enabled (include "gardener-apiserver.hasAdmissionPlugins" .) }}
```

---

Fixed in  621913a